config.yml

```diff
@@ -91,13 +91,29 @@ golang:
   "1.14":
     build_args:
       CI_HELPER_VERSION: *CI_HELPER_VERSION
+      GITLEAKS_VERSION: 4.1.1
+      GOLANGCILINT_VERSION: 1.24.0
+      GOMODUPGRADE_VERSION: 0.1.2
+      GOSWAGGER_VERSION: 0.23.0
+      MIGRATE_VERSION: 4.10.0
+      MOCKGEN_VERSION: 1.4.3
+      MODD_VERSION: 0.8
+      TESTFIXTURES_VERSION: 3.1.1
     test_config:
       cmd:
         - *ci_helper_test
         - aws --version
+        - gitleaks --version
         - go version
+        - golangci-lint --version
+        - migrate -version
+        - mockgen -version
+        - modd --version
+        - swagger version
+        - testfixtures --version
+
     template_vars:
-      BASE_IMAGE_VERSION: 1.14
+      BASE_IMAGE_VERSION: 1.14.2
```

Comment:
https://groups.google.com/forum/#!msg/golang-announce/-9KWN-OUSl0/Jro9NDu8CgAJ :)

Reply:
thanks! waiting for the image release

Reply:
:heavy_check_mark:  upgraded the image